features/support/ui/landing.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'ui'
+
+module UI
+  class Landing < SitePrism::Page
+    set_url '/'
+    set_url_matcher(/wpcc\/\d+\/landing/)
+
+    element :age, '#your_details_form_age'
```

Comment:
@willhall88 @tomas-stefano should we drop the standard if it's inconsistent with the default Rails behaviour. Rails offers a unique identify is it worth using that rather than preceding page object model elements with `.t-`?